datafusion_ray/context.py

```diff
@@ -77,7 +77,7 @@ def _get_worker_inputs(
     plan_bytes = datafusion_ray.serialize_execution_plan(stage.get_execution_plan())
     futures = []
     opt = {}
-    opt["resources"] = {"worker": 1e-3}
+    # opt["resources"] = {"worker": 1e-3}
```

Comment:
I had to remove this; otherwise, the Ray cluster could not find a suitable worker node.

@franklsf95 do we still need this?

Reply:
This only works when you start a Ray cluster with [custom resources](https://docs.ray.io/en/latest/ray-core/scheduling/resources.html#custom-resources), e.g. the head node with `ray start --head --resources='{"head":1}'` and worker nodes each with `ray start --resources='{"worker":1}'`. I had this resource requirement to make sure the tasks run on worker nodes exclusively (for fair benchmarking). If we don't do this, the task could also run on the driver node. Depending on the use case, this may be harmless.